build.gradle

```diff
@@ -405,15 +414,14 @@ subprojects { subproj ->
 			endWithNewline()
 		}
 
-		format 'groovy', {
+		groovy {
 			target 'src/**/*.groovy'
-			indentWithTabs()
```

Comment:
@fvgh Does GrEclipse replace sequences of 4 spaces with tabs? If not, then this line may still be needed.